docker/ubuntu-python3/generate.sh

```diff
@@ -29,7 +29,7 @@ for os_arch in $*; do
 	if [ "$os_arch" != "amd64" -a "$os_arch" != "i386" ]; then
 		echo "ARG qemu_arch=$qemu_arch" >> $df
 		qemu_download_ver=$qemu_ver
-		if [ "$os_arch" = "ppc64le" -a "$qemu_ver" = "v4.0.0" ]; then
+		if [ "$os_arch" = "ppc64le" -a "${qemu_ver:0:6}" = "v4.0.0" ]; then
```

Comment:
it would be nice if you could add some comments to this script as it is not obvious what is happening for which reason here